pkg/util/utils.go

```diff
@@ -35,6 +36,8 @@ const (
 	VSphere                  = "vSphere Kubernetes Cluster"
 )
 
+const VddkConfigPath = "/tmp/config"
```

Comment:
Would you please move this constant definition to the top of this file? Eventually, we might want to move them to a separate file.